HTML/14 altered embellished (JakkOfKlubs).html

```diff
@@ -0,0 +1,194 @@
+									<td><img src = "img/14 Images/Fi.png" style = "width: 60%"></img><br>-F = -15</td>
+									<td><img src = "img/14 Images/Li.png" style = "width: 60%"></img><br>-L = -21</td>
+									<td><img src = "img/14 Images/Ri.png" style = "width: 60%"></img><br>-R = -27</td>
+									<td><img src = "img/14 Images/Xi.png" style = "width: 60%"></img><br>-X = -33</td>
+								</tr>
+								<tr>
+									<td><img src = "img/14 Images/4.png" style = "width: 60%"></img><br>4</td>
+									<td><img src = "img/14 Images/A.png" style = "width: 60%"></img><br>A = 10</td>
+									<td><img src = "img/14 Images/G.png" style = "width: 60%"></img><br>G = 16</td>
+									<td><img src = "img/14 Images/M.png" style = "width: 60%"></img><br>M = 22</td>
+									<td><img src = "img/14 Images/S.png" style = "width: 60%"></img><br>S = 28</td>
+									<td><img src = "img/14 Images/Y.png" style = "width: 60%"></img><br>Y = 34</td>
+								</tr>
+								<tr>
+									<td><img src = "img/14 Images/4i.png" style = "width: 60%"></img><br>-4</td>
+									<td><img src = "img/14 Images/Ai.png" style = "width: 60%"></img><br>-A = -10</td>
+									<td><img src = "img/14 Images/Gi.png" style = "width: 60%"></img><br>-G = -16</td>
+									<td><img src = "img/14 Images/Mi.png" style = "width: 60%"></img><br>-M = -22</td>
+									<td><img src = "img/14 Images/Si.png" style = "width: 60%"></img><br>-S = -28</td>
+									<td><img src = "img/14 Images/Yi.png" style = "width: 60%"></img><br>-Y = -34</td>
+								</tr>
+								<tr>
+									<td><img src = "img/14 Images/5.png" style = "width: 60%"></img><br>5</td>
+									<td><img src = "img/14 Images/B.png" style = "width: 60%"></img><br>B = 11</td>
+									<td><img src = "img/14 Images/H.png" style = "width: 60%"></img><br>H = 17</td>
+									<td><img src = "img/14 Images/N.png" style = "width: 60%"></img><br>N = 23</td>
+									<td><img src = "img/14 Images/T.png" style = "width: 60%"></img><br>T = 29</td>
+									<td><img src = "img/14 Images/Z.png" style = "width: 60%"></img><br>Z = 35</td>
+								</tr>
+								<tr>
+									<td><img src = "img/14 Images/5i.png" style = "width: 60%"></img><br>-5</td>
+									<td><img src = "img/14 Images/Bi.png" style = "width: 60%"></img><br>-B = -11</td>
+									<td><img src = "img/14 Images/Hi.png" style = "width: 60%"></img><br>-H = -17</td>
+									<td><img src = "img/14 Images/Ni.png" style = "width: 60%"></img><br>-N = -23</td>
+									<td><img src = "img/14 Images/Ti.png" style = "width: 60%"></img><br>-T = -29</td>
+									<td><img src = "img/14 Images/Zi.png" style = "width: 60%"></img><br>-Z = -35</td>
+								</tr>
+							</tbody>
+						</table>
+					</span>
+				</div>
+                <div class="page-footer relative-footer">Page 2 of 2</div>
+			</div>
+    </div>
+  </div>
+
+<div id="notification" style="display: none;"></div><div id="optionsMenu">
```

Comment:
this is a very obvious sign that the manual was downloaded from the repo instead of taken from the github, please fix this

Reply:
will do